docs/en/deployment.md

```diff
@@ -2,57 +2,84 @@ Before you launch the solution, review the architecture, supported regions, and
 
 **Time to deploy**: Approximately 10 minutes
 
-## Deployment overview
 
-Use the following steps to deploy this solution on AWS.
-
-* Step1: Check your QuickSight account
-* Step2: Launch the AWS CloudFormation template into your AWS account to deploy the solution.
-* Step3: Update QuickSight permissions
+## Prerequisites
 
+### Sign up for QuickSight
 
 !!! notice
+    You need administrator permission to sign up for QuickSight
 
-    Step1 and Step3 are one time setup per account when deploying this solution for the first time. You don't have to do them when repeating deployment
+* Sign in to the AWS Management Console, navigate to [QuickSight](https://quicksight.aws.amazon.com/).
 
-## Deployment
+* If you did not have a QuickSight account, you need to sign up for QuickSight.
 
-### Step1: Check your QuickSight
+* Choose **Enterprise**, click continue.
 
-* Sign in to the AWS Management Console, navigate to
-[QuickSight](https://quicksight.aws.amazon.com/)
+* In the **Create your QuickSight account** page, fill the necessary information:
 
-* If you did not have a QuickSight account, you need to sign up for QuickSight(you need Administrator permission to sign up, [doc](https://docs.aws.amazon.com/quicksight/latest/user/setting-up-create-iam-user.html)).
+<center>
+![Fill information for QuickSight](./images/create_quicksight.png)
 
-* Choose **Enterprise**, click continue
+Figure 1: Fill information for QuickSight
 
-* In the **Create your QuickSight account** page, fill the necessary information:
+</center>
+
+### Get QuickSight Username (administrator)
+
+* Go to [QuickSight Admin](https://us-east-1.quicksight.aws.amazon.com/sn/admin), record your **QuickSight Username**(not QuickSight account name).
 
 <center>
-![Fill information for quicksight](./images/create_quicksight.png)
+![QuickSight username](./images/quicksight_username.png)
 
-Figure 1: Fill information for quicksight
+
+Figure 2: QuickSight username
 
 </center>
 
-* Go to [quicksight admin](https://us-east-1.quicksight.aws.amazon.com/sn/admin), record your **QuickSight Username**(not QuickSight account name).
+### Get QuickSight Username (non-administrator)
+
+* You need to ask administrator to sign up for QuickSight in your AWS account.
+
+* Navigate to [QuickSight](https://quicksight.aws.amazon.com/), fill your email in below screen:
```

Comment:
any official doc link for this operation?

Reply:
add the link for `sign up for QuickSight`